src/API/Actions/RemoveAttachmentAction.php

```diff
@@ -0,0 +1,23 @@
+<?php declare(strict_types=1);
+
+/*
+ * Created by solutionDrive GmbH
+ *
+ * @copyright 2018 solutionDrive GmbH
+ */
+
+namespace solutionDrive\YellowBox\API\Actions;
+
+class RemoveAttachmentAction extends AbstractAction
+{
+    /** @var string */
+    protected $sRequestType = self::METHOD_DELETE;
+
+    /** @var string */
+    protected $sRequestUrl  = 'attachment/';
```

Comment:
please don't use Hungarian-Notation in this project. I won't comment every single one...